tests/integration/query/simple/with_filter/with_nlike_json_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package simple
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestQuerySimple_WithNotLikeOpOnJSONField_ShouldFilter(t *testing.T) {
+	test := testUtils.TestCase{
+		Actions: []any{
+			testUtils.SchemaUpdate{
+				Schema: `
+					type Users {
+						name: String
+						custom: JSON
+					}
+				`,
+			},
+			testUtils.CreateDoc{
+				DocMap: map[string]any{
+					"name":   "John",
+					"custom": "{\"tree\": \"maple\", \"age\": 250}",
```

Comment:
todo: Please either document why this test appears to be storing a string for this test, or change the test so that it looks more obvious that the json value is a string.

Please apply the same to `TestQuerySimple_WithNotLikeOpOnJSONFieldAllTypes_ShouldFilter` as well.

Reply:
changed to simple strings